src/app/rpmostree-builtin-status.c

```diff
@@ -134,6 +135,16 @@ lookup_array_and_canonicalize (GVariantDict *dict,
   return g_steal_pointer (&ret);
 }
 
+static char*
+vnevra_to_evr (GVariant *vnevra)
```

Comment:
What is the leading `v` here?  Variant?  Yeah must be, I just kept parsing it as `version`.  How about `gv_nevra_to_evr()`?